Zion's End/src/Grid.cpp

```diff
@@ -149,6 +149,21 @@ void Grid::Render(sf::RenderTarget& renderer)
 	}
 }
 
+#pragma warning(push)
+#pragma warning(disable: 4715)
+sf::Vector2f Grid::GetBiblePosition()
```

Comment:
You forgot to change word "two" to "three" on comment edit 😛 
Something like `return {-1, -1} `would do in our case.